BDArmory/Distribution/GameData/BDArmory/Localization/UI/ja.cfg

```diff
@@ -760,8 +760,8 @@ Localization
         #LOC_BDArmory_AIWindow_ExtendAbortTimeContext = 延長中に距離を稼げなかった場合は時間を中止します。
         #LOC_BDArmory_AIWindow_terrainMin = 理想的な機体の向きを実現する回転半径乗数
         #LOC_BDArmory_AIWindow_terrainMax = 反転したクラフトの向きの回転半径乗数
-        #LOC_BDArmory_AIWindow_InvertedTerrainAvoidanceCriticalAngleContext = 逆地形回避またはローリングファーストの臨界角
-        #LOC_BDArmory_AIWindow_TerrainAvoidanceControlSurfaceDeploymentTimeContext = コントロール サーフェスの展開にかかる時間の見積もり
+        ##LOC_BDArmory_AIWindow_TerrainAvoidanceVesselReactionTimeContext = 逆地形回避またはローリングファーストの臨界角

```

Comment:
`#LOC_BDArmory_AIWindow_InvertedTerrainAvoidanceCriticalAngle` should not have been removed here and you have an extra `#` in `#LOC_BDArmory_AIWindow_TerrainAvoidanceVesselReactionTimeContext`.
Looking at google translate,
![Screenshot_20230826_170941](https://github.com/BrettRyland/BDArmory/assets/18440126/cda1850a-e140-4d9d-b706-29f27ad7008a)
I think you've mixed up the tags. This line ought to be
```
#LOC_BDArmory_AIWindow_InvertedTerrainAvoidanceCriticalAngleContext = 逆地形回避またはローリングファーストの臨界角
```
and the line below ought to be something like
```
#LOC_BDArmory_AIWindow_TerrainAvoidanceVesselReactionTimeContext = 最適な旋回のためのセットアップに必要な時間の見積もり
```
(note: I've just used google translate for this, so I'm not sure of the translation). The original explanation for this value was "the time required to deploy the control surfaces", but that got changed to "an estimate of the time required to set up for optimal turning" after we realised that it was more involved than just adjusting control surfaces. It corresponds to the initial period in the diagram for "terrain threat distance" https://github.com/BrettRyland/BDArmory/wiki/2.1-AI-Tuning#terrain-avoidance labelled with "Adjusting control surfaces" where the plane continues in a roughly straight line (slightly S-shaped) before being able to properly turn in a circle.

The short version for the "vessel reaction time" still needs adjusting below on line 816.